apiary-metastore-events/sns-metastore-events/apiary-metastore-listener/pom.xml

```diff
@@ -47,6 +47,15 @@
       <artifactId>slf4j-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
+    </dependency>
+
```

Comment:
Pin `jackson-databind` & `jackson-annotations` to the parent pom version as there was a conflict w/ newer versions of hive.

This occurs in a few other modules under sns-metastore-events

Reply:
I'm sure it does! I'm surprised you didn't also run into more Guava version issues.